bindgen/Cargo.toml

```
@@ -36,7 +36,7 @@ peeking_take_while = "0.1.2"
 prettyplease = { version = "0.2.7", optional = true, features = ["verbatim"] }
 proc-macro2 = { version = "1", default-features = false }
 quote = { version = "1", default-features = false }
-regex = { version = "1.5", default-features = false, features = ["std", "unicode-perl"] }
+regex = { version = "1.5.1", default-features = false, features = ["std", "unicode-perl"] }
```

```suggestion
regex = { version = "1.9.6", default-features = false, features = ["std", "unicode-perl"] }
```

I think it is OK to bump to the latest version supported by bindgen's MSRV (1.60, regex 1.10 is the first to jump to 1.65)